Problem executing ./build within Github actions in devcontainer

Local build is working with:

`
CybICS/software$ ./build.sh 
`


Automated build with Github actions is failing:

`
#8 588.4 make[2]: *** [CMakeFiles/opendnp3.dir/build.make:594: CMakeFiles/opendnp3.dir/cpp/libs/src/opendnp3/gen/CommandPointState.cpp.o] Segmentation fault (core dumped)
#8 588.4 make[2]: *** Waiting for unfinished jobs....
#8 588.5 [ 35%] Building CXX object CMakeFiles/opendnp3.dir/cpp/libs/src/opendnp3/gen/CommandStatus.cpp.o
#8 593.0 make[1]: *** [CMakeFiles/Makefile2:116: CMakeFiles/opendnp3.dir/all] Error 2
#8 593.0 make: *** [Makefile:136: all] Error 2
`

`
#12 10.04 Compiling main program...
#12 10.07 Package libmodbus was not found in the pkg-config search path.
#12 10.07 Perhaps you should add the directory containing `libmodbus.pc'
#12 10.07 to the PKG_CONFIG_PATH environment variable
#12 10.07 No package 'libmodbus' found
#12 11.65 dnp3.cpp:5:10: fatal error: asiodnp3/DNP3Manager.h: No such file or directory
#12 11.65     5 | #include <asiodnp3/DNP3Manager.h>
#12 11.65       |          ^~~~~~~~~~~~~~~~~~~~~~~~
#12 11.65 compilation terminated.
#12 19.46 modbus_master.cpp:31:10: fatal error: modbus.h: No such file or directory
#12 19.46    31 | #include <modbus.h>
#12 19.46       |          ^~~~~~~~~~
#12 19.46 compilation terminated.
#12 23.27 Error compiling C files
#12 23.27 Compilation finished with errors!
#12 ERROR: process "/dev/.buildkit_qemu_emulator /bin/sh -c ./scripts/compile_program.sh 724870.st" did not complete successfully: exit code: 1
------
 > [8/8] RUN ./scripts/compile_program.sh 724870.st:
11.65 dnp3.cpp:5:10: fatal error: asiodnp3/DNP3Manager.h: No such file or directory
11.65     5 | #include <asiodnp3/DNP3Manager.h>
11.65       |          ^~~~~~~~~~~~~~~~~~~~~~~~
11.65 compilation terminated.
19.46 modbus_master.cpp:31:10: fatal error: modbus.h: No such file or directory
19.46    31 | #include <modbus.h>
19.46       |          ^~~~~~~~~~
`

--- a/.github/workflows/devContainer.yml
+++ b/.github/workflows/devContainer.yml
@@ -11,7 +11,7 @@ on:
 jobs:
   raspberry:
     name: Run devcontainer test raspberry
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
    
     steps:
       - name: Login to Docker Hub

